Classes/DDFileLogger+Internal.h

```diff
@@ -18,8 +18,9 @@
 NS_ASSUME_NONNULL_BEGIN
 
 @interface DDFileLogger (Internal)
-- (void)logData:(NSData *)data;
-- (void)writeToFile:(NSData *)data;
+
+- (void)lt_logData:(NSData *)data;
+
 @end
```

Comment:
Saw no reason to have logData calling writeToFile, when writeToFile is not used anywhere else in the project.